sdk/test/resource/resource_test.cc

```diff
@@ -54,6 +55,7 @@ TEST(ResourceTest, create_without_servicename)
       else
         EXPECT_EQ(opentelemetry::nostd::get<std::string>(expected_attributes.find(e.first)->second),
                   opentelemetry::nostd::get<std::string>(e.second));
+    }
```

Comment:
There are a few places in this [test]( https://github.com/open-telemetry/opentelemetry-cpp/blob/4110b56314eb7c3ecbc56b9f6fcb37eb70286858/sdk/test/resource/resource_test.cc#L174 ) using putenv, do you think these can be modified to use setenv? Maybe have a common method here ( https://github.com/open-telemetry/opentelemetry-cpp/blob/main/sdk/include/opentelemetry/sdk/common/env_variables.h ) ?

Reply:
those remaining `putenv`s were correct. To be consistent, I replaced all of them with `sentenv`.